src/main/java/cz/brmlab/yodaqa/flow/dashboard/QuestionAnswer.java

```diff
@@ -11,24 +11,19 @@
 	protected String text;
 	protected double confidence;
 	protected int ID;
-	protected String source;
-	protected List<Integer> passageIDs= new ArrayList<>();
+	protected List<Integer> snippetIDs= new ArrayList<>();
 	public QuestionAnswer(String text, double confidence) {
 		this.text = text;
 		this.confidence = confidence;
 	}
-	public List<Integer> getPassageIDList() {return passageIDs;}
-	public void addToPassageList(int passageID){
-		passageIDs.add(passageID);
+	public List<Integer> getSnippetIDs() {return snippetIDs;}
+	public void addToSnippetIDList(int snippetID){
+		snippetIDs.add(snippetID);
 	}
-	public List<Integer> getPassageList(){
-		return passageIDs;
-	}
-	/** @return the text */
+ 	/** @return the text */
 	public String getText() { return text;}
 	/** @return the confidence */
 	public double getConfidence() { return confidence; }
 	public void setID(int ID) {this.ID = ID;}
```

Comment:
I think the original idea is that we can get even more extra details for a specific answer via the API than we send by default.

We don't use this now, but I suspect that we will need to, sooner or later, as we add more details to the answer.  At any rate, I think we should either do it properly or remove the code again for now.